src/main/java/org/fundacionjala/coding/ketty/Ean.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.coding.ketty;
+
+/**
+ * verifica que un string  EAN.
+ */
+public class Ean {
+
+
+    /**
+     * El  CheckSum, valida si string numero es "EAN".
+     *
+     * @param num es la es una cadena numerica.
+     * @return un valor boolean,  true si la cadena numerica es "EAN" y false cuando no es "EAN".
+     */
+    public boolean checkSum(String num) {
+        final int numeroImpar = 1;
+        final int numeroPar = 3;
+        int index = 1;
+        char numeroChar = ' ';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "numeroChar". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)